src/routes/api-documentation.ts

```diff
@@ -21,7 +21,7 @@ export default (): AppMiddleware => (
           '@type': hydra.Class,
           [hydra.title]: { '@value': 'API entry point', '@language': 'en' },
           [hydra.supportedOperation]: {
-            '@type': hydra.Operation,
+            '@type': [hydra.Operation, schema.ViewAction],
```

Comment:
Not sure if `ViewAction` is the most appropriate. `DownloadAction`? `ReceiveAction`?

Reply:
Though thinking maybe there's really no need to include `GET` operations, as Hydra resources _have_ to be dereferenceable?

Reply:
What about [`ReadAction`](https://schema.org/ReadAction)?